tests/integration/profiling/rule_based_profiler/test_run_profiler_notebook.py

```diff
@@ -246,3 +246,45 @@ def test_run_multibatch_configured_asset_example(tmp_path):
             "great_expectations/uncommitted/validations/example_configured_suite",
         ]
         clean_up_test_files(base_dir=base_dir, paths=paths_to_clean_up)
+
+
+def test_run_multibatch_sql_asset_example(tmp_path):
+    """
+    What does this test and why?
+    One of the resources we have for multi-batch Expectations is a Jupyter notebook that explains/shows the components
+    in code. This test ensures the codepaths and examples described in the Notebook actually run and pass, nbconvert's
+    `preprocess` function.
+    """
+    base_dir: str = file_relative_path(
+        __file__, "../../../test_fixtures/rule_based_profiler/example_notebooks"
+    )
+    notebook_path: str = os.path.join(base_dir, "MultiBatchExample_SqlExample.ipynb")
+    # temporary output notebook for traceback and debugging
+    output_notebook_path: str = os.path.join(
+        tmp_path, "MultiBatchExample_SqlExample_executed.ipynb"
+    )
+    with open(notebook_path) as f:
+        nb: NotebookNode = nbformat.read(f, as_version=4)
+
+    ep: ExecutePreprocessor = ExecutePreprocessor(timeout=60, kernel_name="python3")
+
+    try:
+        ep.preprocess(nb, {"metadata": {"path": base_dir}})
+    except CellExecutionError:
+        msg: str = f'Error executing the notebook "{notebook_path}".\n\n'
+        msg += f'See notebook "{output_notebook_path}" for the traceback.'
+        print(msg)
```

Comment:
Another option is to put `msg` in an exception and do exception chaining. 
```
except CellExecutionError as e:
    msg = ...
    raise MyException(msg) from e
```
See https://docs.python.org/3/tutorial/errors.html#exception-chaining

The reason to do this is the print msg may get lost in the error output. It will likely be send to STDOUT and could get routed away from where STDERR is being routed.  Exception chaining preserves the original exception and backtrace and will print out the new one underneath.

Reply:
Oo thank you @billdirks . I'll make note of this for future as well